addon/mixins/taggable-mixin.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Mixin.create({
             Ember.A(currentTags);
```

Comment:
When I wrote this, I was having a hard time getting changedAttributes to update. I first did `file.get('tags').pushObject(tag)`, but it was saying that pushObject was not a function, so I changed it to Ember.A(file.get('tags')).pushObject(tag). But this results in changedAttributes being an empty object. So no PATCH request was sent.

The only way I could get changedAttributes to modify was to copy the current tags (not referencing original) and push new tag onto list, and then set tags equal to this list.  

Sam said that Henrique's work may fix this needing a roundabout way to modify the tags.